docs/roadmap.md

```diff
@@ -11,37 +11,49 @@ Here's how we intend to roll out Project Reunion:
 <table>
   <tbody>
     <tr>
-      <th>//BUILD 2020</th>
-      <th>End of 2020</th>
-      <th>2021++</th>
+      <th>Reunion 0.5 (March 2021)</th>
+      <th>Reunion 0.8 (May 2021)</th>
+      <th>Reunion 1.0 (October 2021)</th>
+      <th>Past October Release</th>
     </tr>
     <tr>
-      <td>
+    <td>
+      <ul>
+            <li>1st supported public release</li>
+            <li>UWP features available to unpackaged Win32</li>
```

Comment:
Isn't essentially only WinUI available, but other than that, what UWP features are really available in 0.5? I wouldn't consider AppLifecycle one of those as currently AppLifecycle is only exposing things Win32 apps could already do (just in a different API)... it hasn't added Share/Toasts/etc